packages/ckeditor5-widget/src/widgetresize.js

```diff
@@ -87,7 +87,7 @@ export default class WidgetResize extends Plugin {
 			}
 		};
 
-		const redrawFocusedResizerThrottled = throttle( redrawFocusedResizer, 200 ); // 5fps
+		const redrawFocusedResizerThrottled = throttle( redrawFocusedResizer, 50 ); // 5fps
```

Comment:
FPS changed:

```suggestion
		const redrawFocusedResizerThrottled = throttle( redrawFocusedResizer, 50 ); // 20fps
```

Reply:
How did you measure this?

Reply:
I've just updated the value in the comment to match `T = 50ms` - FPS should grow 4 times.

Reply:
If the comment itself is rubbish we can remove it.

Reply:
I'm gonna remove it :D